sycl/include/sycl/ext/oneapi/experimental/root_group.hpp

```diff
@@ -91,7 +92,8 @@ template <int Dimensions> sycl::sub_group get_child_group(group<Dimensions> g) {
 
 namespace this_kernel {
 template <int Dimensions> root_group<Dimensions> get_root_group() {
-  return this_nd_item<Dimensions>().ext_oneapi_get_root_group();
+  return sycl::ext::oneapi::this_work_item::this_nd_item<Dimensions>()
+      .ext_oneapi_get_root_group();
 }
 } // namespace this_kernel
```

Comment:
Just a note: We should update the `root_group` extension, too.  This function should be defined as `this_work_item::get_root_group()` to match the others.